comp/venue/signage.yaml

```
@@ -1,6 +1,6 @@
 summary: Prepare signage for the $SRYYYY Competition
 
-priority: major
+priority: should
```

There are H&S implications with this, so it might need to be:

```suggestion
priority: must
```